specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -239,147 +233,142 @@ The well-known
+| GUID_SESSION_USER_PRESENCE        | The user status associated with the application's session has changed. | `UserPresenceStatus`                                      |
+| GUID_SYSTEM_AWAYMODE              | The system is entering or exiting away mode.                 | _Not implemented_                                         |
+
+In addition, the PowerRegisterForEffectivePowerModeNotifications API is brought forward with a new property and event:
+
+| Existing API                                             | Description                             | New property and event                          |
+| -------------------------------------------------------- | --------------------------------------- | ----------------------------------------------- |
+| PowerRegisterForEffectivePowerModeNotifications function | The effective power mode of the system. | `EffectivePowerMode, EffectivePowerModeChanged` |
 
 The existing PowerManager class exposes only static properties and events: all members are valid to
 be brought over to the Reunion version. Many existing enumerated values are brought over as-is as
 well.
 
 ```idl
-namespace Microsoft.Windows.System.Power
+namespace Microsoft.ProjectReunion
 {
-    // Based on Windows.System.Power.BatteryStatus
-    enum BatteryState {
-        NotPresent,
-        Discharging,
-        Idle,
-        Charging
-    }
-
-    // Based on Windows.System.Power.EnergySaverStatus
-    enum EnergySaverState {
+    // Enums duplicated from the existing WinRT PowerManager API.
+    enum EnergySaverStatus
+    {
+        Uninitialized = 0,
         Disabled,
         Off,
         On
-    }
+    };
 
-    // Based on Windows.System.Power.PowerSupplyStatus
-    enum PowerSupplyState {
-        NotPresent,
+    enum BatteryStatus
+    {
+        NotPresent = 0,
+        Discharging,
+        Idle,
+        Charging
+    };
+
+    enum PowerSupplyStatus
+    {
+        NotPresent = 0,
         Inadequate,
         Adequate
-    }
+    };
 
-    // Kind of the device's power source
-    enum PowerSourceKind {
-        AC,
-        DC,
-        ShortTerm
-    }
+    // Enums based on the Win32 PowerSettingRegisterNotification API.
+    enum PowerSourceKind
+    {
+        AC = 0,
+        DC
+    };
 
-    // State of the primary display
-    enum DisplayState {
-        Off,
+    enum DisplayStatus
+    {
+        Off = 0,
         On,
         Dimmed
-    }
+    };
 
-    // Indicates whether the system is busy or idle.
-    enum SystemBusyState {
-        Idle,
-        Busy,
-    }
-
-    // Active power management personalities supported by the system
-    enum PowerSchemePersonality {
+    enum EffectivePowerMode
+    {
+        BatterySaver,
+        BetterBattery,
+        Balanced,
         HighPerformance,
-        PowerSaver,
-        Balanced
-    }
+        MaxPerformance, // v1 last supported
```

Comment:
I made these changes to my PR too - we should change the comment to something clearer like 
`    enum EffectivePowerMode
    {
        // EFFECTIVE_POWER_MODE_V1 values
        BatterySaver,
        BetterBattery,
        Balanced,
        HighPerformance,
        MaxPerformance,
        // EFFECTIVE_POWER_MODE_V2 additional values
        GameMode,
        MixedReality
    };`
    Open to something even better.

Reply:
Fixed